Scripts/main.js

```diff
@@ -1,15 +1,30 @@
-var brdfViewport = null;
-var modelViewport = null;
-
-var render = function(time) {
-  brdfViewport.render(time);
-  modelViewport.render(time);
-  requestAnimationFrame(render);
-}
-
-/* Requires BRDFViewport.js */
-document.addEventListener('DOMContentLoaded', function () {
-  brdfViewport = new BRDFViewport("brdf-canvas", 512, 512);
-  modelViewport = new ModelViewport("model-canvas", 512, 512);
-  requestAnimationFrame(render);
-});
+"use strict";
+
+import BRDFViewport from "./BRDFViewport.js";
+import ModelViewport from "./ModelViewport.js";
+import ControlsManager from "./ControlsManager.js";
+
+//Wrap everything in a function so that we do not pollute
+//the global namespace.
```

Comment:
I don't mind polluting the global namespace a bit. I actually prefer having access to these objects from the javascript console, and would prefer they be global.

Just want to avoid polluting when collisions will inevitably happen, like when declaring a variable "gl" as a global.